File: Tests/server/server_executors/executors/LocalizationExecutor.cpp
#include "LocalizationExecutor.h"

#include <Utility/WebFrameworkLocalization.hpp>

void LocalizationExecutor::doGet(framework::HTTPRequest& request, framework::HTTPResponse& response)
{
	response.setBody
	(
		framework::JSONBuilder().append
		(
			"result",
			framework::utility::getLocalizedString("LocalizationData", "key", request.getJSON().get<std::string>("language"))
		)
	);
}

DECLARE_EXECUTOR(LocalizationExecutor)

File: Tests/executors/CSharp/CoreExecutors/LocalizationExecutor.cs
using Framework;
using Framework.Utility;

public class LocalizationExecutor : StatelessExecutor
{
	public override void DoGet(HttpRequest request, HttpResponse response)
	{
		response.SetBody
		(
			new
			{
				result = WebFrameworkLocalization.GetLocalizedString("LocalizationData", "key", request.GetJson()["language"].GetString()!)
			}
		);
	}
}
